sentry-android-core/build.gradle.kts

```diff
@@ -24,10 +24,9 @@ android {
     }
 
     buildTypes {
-        getByName("debug") {
-            isMinifyEnabled = false
-        }
+        getByName("debug")
         getByName("release") {
+//            isMinifyEnabled = true // Do we want to enable it for our lib?
```

Comment:
Does it really make sense? I mean, it is an open source project and the source code is available online. Also you would have to carefully craft your proguard file (in terms of code roots) to make sure code that is used only when integrated into a consumer app does not get deleted by minification.
I don't think it is worth the pain.

Reply:
@ninniuz I agree

Reply:
Also agreed. But don't we need to proguard file anyway? To be brought into the app and included when the app is minified?

Reply:
this is already being done by https://github.com/getsentry/sentry-android/blob/587ed63fb465e11b9ee6271eaeccf0422f21f7be/sentry-android-core/build.gradle.kts#L31